tfhe/src/integer/server_key/radix_parallel/tests.rs

```diff
@@ -1813,9 +1814,8 @@ where
     P: Into<PBSParameters>,
 {
     let (cks, mut sks) = KEY_CACHE.get_from_params(param);
-    let cks = RadixClientKey::from((cks, NB_CTXT));
-
     sks.set_deterministic_pbs_execution(true);
+    let cks = RadixClientKey::from((cks, NB_CTXT));
```

Comment:
is that required or just a rebase result ?

Reply:
result of failed rebase

Reply:
fixed